src/prefect/server/api/deployments.py

```diff
@@ -192,6 +192,39 @@ async def update_deployment(
                 status.HTTP_404_NOT_FOUND, detail="Deployment not found."
             )
 
+        schedules_to_patch: list[schemas.actions.DeploymentScheduleUpdate] = []
+        schedules_to_create: list[schemas.actions.DeploymentScheduleUpdate] = []
+        all_provided_schedules_have_slugs = deployment.schedules and all(
```

Comment:
this isn't necessary, but in case it resonates with you: you could consider shortening this booleans to `all_provided_have_slugs` and `all_existing_have_slugs` to shorten the lines; their meaning should still be clear in context